usagov/vote-gov-drupal
# Feature/vote 2766 nvrf update

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-2766

## Description

Various updates to nvrf integration including updating nvrf version to latest release, updating path to the nvrf form, removing unused fields.

## Deployment and testing

### Post-deploy steps

1. cd into vote_nvrf module and run `npm i` and `npm run build`
2. `lando retune`

### QA/Testing instructions

1. visit http://vote-gov.lndo.site/admin/content/block/3?destination=/admin/content/block and add a link label value in the digital form accordion.
2. visit http://vote-gov.lndo.site/register/washington click on the button for form filler
3. make sure the form loads at http://vote-gov.lndo.site/register/washington/mail-in-form-filler
4. test traversing the whole form to completion

## Checklist for the Developer




## Checklist for the Peer Reviewers
